doc/nrf/gs_ins_windows.rst

```diff
@@ -143,6 +143,8 @@ To clone the repositories, complete the following steps:
         * West was introduced after |NCS| v0.3.0.
           Therefore, you cannot use it to check out v0.1.0 or v0.3.0.
 
+        * *NCS_version* can be a branch (eg. ``master``), a tag (eg. ``v1.0.0``), or even a SHA (eg. ``224bee9055d986fe2677149b8cbda0ff10650a6e``)
```

Comment:
This doesn't make sense, because we're in the option for checking out a tag and already stated that NCS_version should be replaced with a tag, and the next option states how to check out master.
You could add a third option (in addition to checking out a release and checking out the latest state of development) explaining how to check out a branch or a SHA.

Reply:
I think that it is overkill to have 3 options there with almost identical instructions. Maybe I can try to rewrite this so that it covers all 3 with a single text base.

Reply:
Maybe - but keep in mind that our main audience are users that work with a specific release. This is what we want people to do. Checking out master, a feature branch, or a SHA is only interesting for advanced users.